packages/salesforcedx-apex-debugger/src/messages/i18n.ts

```diff
@@ -36,5 +36,7 @@ export const messages = {
   idle_warn_text:
     'You have been idle for %s minutes. To prevent your debugger session from being terminated, run or step through code, or inspect your variables, in the next %s minutes.',
   idle_terminated_text:
-    'Your debugger session is being terminated because it has been idle for %s minutes.'
+    'Your debugger session is being terminated because it has been idle for %s minutes.',
+  invalid_isv_project_config:
+    'Your project configuration is invalid/incomplete for ISV debugging. Create a new ISV project and try again.'
```

Comment:
Would they create the new project by returning to Setup? If so, maybe this would work?

'Your project configuration is invalid or incomplete for ISV debugging. Return to the Apex Debugger page in Setup, start a new partner debugging session, and try again.'

If this isn't the right way to create a new project, how do they do it?

Reply:
I think this will work nicely. I'll incorporate it.